styles.css

```
@@ -11480,3 +11480,32 @@ body:has(#side-panel.active) #close-fullscreen-cover-btn {
     opacity: 0.8;
     vertical-align: middle;
 }
+
+img.cd {
+    border-radius: 50% !important;
+    box-shadow: none;
+    clip-path: url(#cd-hole-clip);
```

Stylelint (17.7.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)